pkl/project_test.go

```diff
@@ -171,3 +196,38 @@ func TestLoadProject(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadProjectWithProxy(t *testing.T) {
+	manager := NewEvaluatorManager()
+	version, err := manager.(*evaluatorManager).getVersion()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if pklVersion0_26.isGreaterThan(version) {
```

Comment:
I had to hoist this file into the `pkl` package to access this private member, hence most of the diff in the existing test.